app/views/__init__.py

```diff
@@ -50,64 +100,21 @@ def register(app):
         :param app: a flask app instance
         :return:
         """
-        app.register_blueprint(pages)
-
         app.register_blueprint(home_routes)
-        app.register_blueprint(utils_routes)
-        app.register_blueprint(sitemaps)
-
-        app.register_blueprint(babel)
-        app.register_blueprint(event_invoicing)
-        #app.register_blueprint(explore)
-        app.register_blueprint(ticketing)
-        app.register_blueprint(event_detail)
-
-        # Blueprints that require login
-        app.register_blueprint(events)
-        app.register_blueprint(event_speakers)
-        app.register_blueprint(event_sessions)
-        app.register_blueprint(event_scheduler)
-        app.register_blueprint(event_export)
-        app.register_blueprint(event_roles)
-        app.register_blueprint(event_invites)
-        app.register_blueprint(event_ticket_sales)
-        app.register_blueprint(event_sponsors)
-
-        app.register_blueprint(my_tickets)
-        app.register_blueprint(my_sessions)
-        app.register_blueprint(settings)
-        app.register_blueprint(notifications)
-        app.register_blueprint(profile)
-
-        # Blueprints that are accessible only by a super admin
-        app.register_blueprint(sadmin)
-        app.register_blueprint(sadmin_events)
-        app.register_blueprint(sadmin_sales)
-        app.register_blueprint(sadmin_sessions)
-        app.register_blueprint(sadmin_users)
-        app.register_blueprint(sadmin_permissions)
-        app.register_blueprint(sadmin_messages)
-        app.register_blueprint(sadmin_reports)
-        app.register_blueprint(sadmin_settings)
-        app.register_blueprint(sadmin_modules)
-        app.register_blueprint(sadmin_content)
-        app.register_blueprint(sadmin_debug)
-
-
-@event_speakers.before_request
-@event_sessions.before_request
-@event_scheduler.before_request
-@event_export.before_request
-@events.before_request
-@event_roles.before_request
-@event_sponsors.before_request
-@event_invites.before_request
-@event_ticket_sales.before_request
-@my_tickets.before_request
-@my_sessions.before_request
-@settings.before_request
-@notifications.before_request
-@profile.before_request
-def check_accessible():
-    if not AuthManager.is_accessible():
-        return redirect(url_for('admin.login_view', next=request.url))
+        admin = Admin(app, name='admin', template_mode='bootstrap3', index_view=MyAdminIndexView(),
+                      base_template='my_master.html')
+
+        # Get all the models in the db, all models should have a explicit __tablename__
+        classes, models, table_names = [], [], []
+        for clazz in db.Model._decl_class_registry.values():
```

Comment:
using class would shadow built-in `class`, using `class_` now instead of `clazz` now